src/components/cropper/Cropper.vue

```diff
@@ -87,7 +87,7 @@
 
       <p-button
         data-testid="cropper-reset"
-        class="flex-shrink-0"
```

Comment:
please move to custom style, because using tailwind class directly in class attribute, it not recognize when published